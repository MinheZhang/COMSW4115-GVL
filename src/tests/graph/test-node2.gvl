int main() {
  node n;
  n = create_node(0.0, 1.0, 1.0, 255, 250, 245, 0);
  printf(get_node_x(n));
  printf(get_node_y(n));
  printf(get_node_radius(n));
  printi(get_node_r(n));
  printi(get_node_g(n));
  printi(get_node_b(n));
  return 0;
}