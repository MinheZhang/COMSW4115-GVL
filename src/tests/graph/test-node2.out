0
1
1
255
250
245